### ISSUE TITLE
React Developer Tools add failed

### ISSUE THREAD

**Issue by tonywxx** [electron-vite/electron-vite-react#207]
#### Describe the problem you confuse

any help?

**Comment by RSS1102** [comment]
Are you using the React Google Chrome plugin within Electron?

**Comment by Stanzilla** [comment]
> Are you using the React Google Chrome plugin within Electron?

Yeah that's the only way atm. I have the same issue, trying to install react and redux devtools. It works in production/packaged app but not in dev mode. 

@caoxiemeihao any ideas?

**Comment by Stanzilla** [comment]
@caoxiemeihao any chance you could take a look at this?

**Comment by Imaginativeone** [comment]
Is there a way to offer a bounty for resolving this issue? I'll start the bids with $100.

**Comment by caoxiemeihao** [comment]
> @caoxiemeihao any chance you could take a look at this?

@Stanzilla Sorry, I just saw this message. I've been very busy at work recently. I'll take some time to look at this issue.

**Comment by Stanzilla** [comment]
> > @caoxiemeihao any chance you could take a look at this?
> 
> @Stanzilla Sorry, I just saw this message. I've been very busy at work recently. I'll take some time to look at this issue.

Awesome, thank you! Glad that you found a new job as well, I remember you were looking!

**Comment by caoxiemeihao** [comment]
@Stanzilla  I noticed your comments about this issue elsewhere. Maybe we need to wait for an official response from Electron.

**Comment by Stanzilla** [comment]
> @Stanzilla  I noticed your comments about this issue elsewhere. Maybe we need to wait for an official response from Electron.

I'm not sure since it does work in build mode my guess is that vite just does not know where to look for the files in dev mode.

**Comment by Imaginativeone** [comment]
> @Stanzilla I noticed your comments about this issue elsewhere. Maybe we need to wait for an official response from Electron.

Is there a way to expedite an official response from Electron?

**Comment by silyont** [comment]
To everyone got struggled:

I found a workaround to add back react devtools to electron. The root cause is the default startup argument `--no-sandbox` in `electron-vite-plugin`. Therefore I tried to modify `vite.config.mts` like this:

```typescript
electron([
  {
    entry: 'electron/main.ts',
    onstart: (option) => option.startup(['.']),
    // ... other configs
  },
  // ... other configs
])
```

**Comment by Stanzilla** [comment]
> To everyone got struggled:
> 
> I found a workaround to add back react devtools to electron. The root cause is the default startup argument `--no-sandbox` in `electron-vite-plugin`. Therefore I tried to modify `vite.config.mts` like this:
> 
> ```ts
> electron([
>   {
>     entry: 'electron/main.ts',
>     onstart: (option) => option.startup(['.']),
>     // ... other configs
>   },
>   // ... other configs
> ])
> ```

Oh my god, it works, thank you!
